src/controllers/highlights/HighlightController.cpp

```diff
@@ -216,11 +216,9 @@ void rebuildBadgeHighlights(Settings &settings,
                         }
 
                         return HighlightResult{
-                            highlight.hasAlert(),
-                            highlight.hasSound(),
-                            highlightSoundUrl,
-                            highlight.getColor(),
-                            false,  // showInMentions
+                            highlight.hasAlert(),       highlight.hasSound(),
+                            highlightSoundUrl,          highlight.getColor(),
+                            highlight.showInMentions(),
```

Comment:
probably reads better as the old implementation? the formatter wanted this but i don't really like it
```suggestion
                            // clang-format off
                            highlight.hasAlert(),
                            highlight.hasSound(),
                            highlightSoundUrl,
                            highlight.getColor(),
                            highlight.showInMentions(),
                            // clang-format on
```